src/index.html

```diff
@@ -1,13 +1,458 @@
+                  src="./img/men-near-the-painting.png"
+                  alt="men-near-the-painting"
+                  class="news__photo"
+                >
+
+                <h4 class="news__title">Virtual group tours</h4>
+                <p class="news__description">
+                  Request a guided tour, reserve admission
+                  and lead your own small group, or request
+                  a live virtual tour led by a Museum guide.
+                </p>
+              </article>
+            </div>
+          </section>
+          <section class="contact-us" id="contact-us">
+            <h2 class="section-title contact-us__title">Contact us</h2>
+            <div class="information contact-us__information">
+              <div
+                class="information__block
+                information__block--text"
+              >
+                <h4 class="information__title">The Met Fifth Avenue</h4>
+                <p>Phone</p>
+                <a
+                  href="tel: +1 212-535-7710"
+                  class="information__bold"
+                >
+                  +1 212-535-7710
+                </a>
+
+                <p>Address</p>
+                <a
+                  href="https://www.google.com/maps?q=48.8588443,2.2943506"
+                  target="_blank"
+                  class="information__bold"
+                >
+                  1000 Fifth Avenue<br>New York, NY 10028
+                </a>
+
+
+              </div>
+              <div
+                class="information__block
+                information__block--text
+                information__block--text--middle"
+              >
+                <h4 class="information__title">The Met Cloisters</h4>
+                <p>Phone</p>
+                <a
+                  href="tel: +1 212-535-7710"
+                  class="information__bold">
+                    +1 212-923-3700
+                </a>
+                <p>Address</p>
+
+                <a
+                  href="https://www.google.com/maps?q=48.8588443,2.2943506"
+                  target="_blank"
+                  class="information__bold"
+                >
+                  99 Margaret Corbin Drive
+                  <br>Fort Tryon Park
+                  <br>New York, NY 10040
+                </a>
+              </div>
+
+              <div
+                class="information__block
+                information__block--form"
+              >
+                <form
+                  action="post"
+                  class="form"
+                  onsubmit="this.reset(); return false;"
+                >
+                  <input
+                    type="text"
+                    placeholder="Name"
+                    class="form__item"
+                  >
+
+                  <input
+                    type="email"
+                    placeholder="Email"
+                    class="form__item"
+                  >
+
+                  <input
```

Comment:
In my opinion, it is better to use `<textarea></textarea>` here because the text does not wrap on a new line.

![image](https://github.com/mate-academy/layout_landing-page/assets/108331415/ee4a680f-40a1-4f8d-ad1a-ded653d58eee)